Source/SvgElementFactory.cs

```diff
@@ -186,7 +186,7 @@ private void SetAttributes(SvgElement element, XmlReader reader, SvgDocument doc
                         var inlineSheet = stylesheetParser.Parse("#a{" + reader.Value + "}");
                         foreach (var rule in inlineSheet.StyleRules)
                             foreach (var declaration in rule.Style)
-                                element.AddStyle(declaration.Name, declaration.Value, SvgElement.StyleSpecificity_InlineStyle);
+                                element.AddStyle(declaration.Name, declaration.Original, SvgElement.StyleSpecificity_InlineStyle);

```

Comment:
Yes this is the solution, so that the values don't change because before the update .Value was the same as .Original